ui/app/styles/core/select.scss

```diff
@@ -2,7 +2,103 @@
  * Copyright (c) HashiCorp, Inc.
  * SPDX-License-Identifier: MPL-2.0
  */
+select {
+  width: 100%;
+  appearance: none;
+}
 
 .select select {
+  background-color: $ui-gray-050;
+  box-shadow: 0 3px 1px rgba($black, 0.12);
+  border-color: $grey-light;
+  border-radius: 2px;
+  color: $grey-dark;
+  cursor: pointer;
+  display: block;
+  font-size: 1em;
+  height: 2.5rem;
+  max-width: 100%;
+  outline: none;
+  padding-bottom: calc(0.375em - 1px);
+  padding-left: $size-8;
+  padding-right: $size-8;
+  padding-top: calc(0.375em - 1px);
   text-rendering: auto !important;
+
+  @include until($desktop) {
+    font-size: 16px;
+  }
+
+  &::placeholder {
+    opacity: 0.5;
+  }
+
+  &:active,
+  &:focus,
+  &:focus-visible,
+  &.is-active {
+    border-color: $blue-500;
+    box-shadow: 0 0 0 0.125em rgba(21, 99, 255, 0.25);
+  }
+
+  .has-background-grey-lighter & {
+    background-color: $white;
+  }
+}
+
+// .select select:not([multiple]) {
+//   padding-right: 2.5em;
+// }
+
+// .select select[multiple] {
+//   height: auto;
+//   padding: 0;
+// }
+
+// .select select[multiple] option {
+//   padding: 0.5em 1em;
+// }
+
+// .select.is-fullwidth {
+//   width: 100%;
+// }
+
+// .select.is-fullwidth select {
+//   width: 100%;
+// }
+
+.select select[disabled] {
```

Comment:
![image](https://user-images.githubusercontent.com/6618863/227731993-ec3118bd-2c72-4658-a780-70c9aab504e1.png)

here's a select disabled for reference above a select. There are not a lot of select disabled out there.